.gitignore

```diff
@@ -1,3 +1,9 @@
 node_modules
 npm-debug.log
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+## generated files
+ng-inspector.firefox/*.xpi
+ng-inspector.firefox/data/icons/
+ng-inspector.firefox/data/ng-inspector.js
```

Comment:
May want to double-check with @rev087 on ignoring these files. The current pattern for this repository has been to check-in the built files.